spring-batch-docs/asciidoc/whatsnew.adoc

```diff
@@ -124,6 +124,55 @@ that uses these new APIs in the
 link:$$https://github.com/spring-projects/spring-batch/tree/master/spring-batch-samples#remote-chunking-sample$$[samples module]
 as well as more details in the <<spring-batch-integration.adoc#remote-chunking,Spring Batch Integration>> chapter.
 
+Just like the remote chunking configuration simplification, this version also
+introduces new APIs to simplify a remote partitioning setup:
+`RemotePartitioningMasterStepBuilder` and `RemotePartitioningWorkerStepBuilder`.
+Those can be autowired in your configuration class if the
+`@EnableBatchIntegration` is present as shown in the following example:
+
+[source, java]
```

Comment:
I don't find it is pretty useful to mention the whole sample in the `What's New`: I might just not be interested in the future you are describing. So, the simple link to the target chapter with details is really enough.